### PR TITLE
♻️ Restructure to a Gateway with a UI & API above Core

### PR DESCRIPTION
**Pull request by Hona**
image

### CODE REVIEW

File: src/TimesheetGPT.WebUI/TimesheetGPT.WebUI.csproj [hunk 1]
```diff
@@ -37,5 +44,14 @@
...
 
+  <ItemGroup>
+    <None Remove="Properties\launchSettings.json" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <UpToDateCheckInput Remove="Pages\Index.razor" />
+    <UpToDateCheckInput Remove="Pages\Error.cshtml" />
+    <UpToDateCheckInput Remove="Pages\_Host.cshtml" />
+  </ItemGroup>
```
Comment (bradystroud) [inline src/TimesheetGPT.WebUI/TimesheetGPT.WebUI.csproj:55]: 🤷🏻‍♂️

File: .github/workflows/codeql.yml
Comment (bradystroud) [inline .github/workflows/codeql.yml:1]: 🚮

**Comment by SamBWagner** [comment]
Build failing. Fix it.

File: .github/workflows/codeql.yml
Comment (SamBWagner) [inline .github/workflows/codeql.yml:1]: don't add CodeQL

**Review by SamBWagner — CHANGES_REQUESTED** [review]

**Review by bradystroud — APPROVED** [review]

**Review by SamBWagner — APPROVED** [review]
lgtm

**Pull request merged**